x-pack/test/functional/apps/ml/short_tests/model_management/model_list.ts

```diff
@@ -144,6 +156,26 @@ export default function ({ getService }: FtrProviderContext) {
           { pipelineName: `pipeline_${modelWithPipelineData.modelId}`, expectDefinition: false },
         ]);
       });
+
+      describe('Add trained model flyout', () => {
+        it('should display elements on Manual Download tab correctly', async () => {
+          await ml.testExecution.logTestStep('Open the Add Trained Model Flyout');
+          await maybeClearSearchInput();
```

Comment:
why do we need to clear the search input?

Reply:
Again, in my flaky test runs I noticed what I presume to be leftover state from another it block.  What I observed is that the search box had a search term within it, which changes the display's contents.  My assumption is that somehow this was blocking the opening of the flyout.

Reply:
shall we place it in `beforeEach` (or `afterEach`) block?